feat: bring back blinking cursor indicating activity

xjose97x opened the pull request:
This should fix: https://github.com/Chainlit/chainlit/issues/2376

asvishnyakov reviewed (changes requested):
@xjose97x Thank you for the contribution! Nice catch! However
1. e2e tests doesn't test all conditions specified in #2393
2. `cy.wait` proven to cause issues and worse e2e tests stability. Please try to wait for something else - I believe simple reordering (check text / length first) should work

xjose97x commented:
> @xjose97x Thank you for the contribution! Nice catch! However
> 
> 1. e2e tests doesn't test all conditions specified in #2393
> 2. `cy.wait` proven to cause issues and worse e2e tests stability. Please try to wait for something else - I believe simple reordering (check text / length first) should work

Thanks @asvishnyakov .
Just added to test also Step (previously I was only testing Message).
Am I missing other conditions? Sorry I looked at that PR but not sure what conditions you are referring to.

I also removed cy.wait and now I'm just using timeouts.
